lib/HTTP/Status.pm

```diff
@@ -121,7 +122,9 @@ die if $@;
 push(@EXPORT, "RC_MOVED_TEMPORARILY");
 
 my %compat = (
```

Comment:
I'm afraid that with the introduction of `306` and `418` both being `(Unused)` we end up in an interesting ambiguity that we may need to tackle some how. But we definitely need to have `HTTP_I_AM_A_TEAPOT` to resolve to `418`